token.go

```diff
@@ -46,6 +46,6 @@ type Token struct {
 
 // PIIParams are parameters for personal identifiable information (PII).
 type PIIParams struct {
-	Params           `form:"*"`
-	PersonalIDNumber *string `form:"personal_id_number"`
+	Params   `form:"*"`
+	IDNumber *string `form:"id_number"`
```

Comment:
This is a breaking change and likely requires a major version to play it safe even if it likely did not work.